src/test/java/com/wikia/webdriver/common/core/purge/PurgeMethod.java

```diff
@@ -1,27 +1,21 @@
-/**
- *
- */
 package com.wikia.webdriver.common.core.purge;
 
-import org.apache.commons.httpclient.HttpMethodBase;
+import java.net.URI;
+
+import org.apache.http.client.methods.HttpRequestBase;
 
 /**
  * @author Karol 'kkarolk' Kujawiak
  */
-public class PurgeMethod extends HttpMethodBase {
-
-  public PurgeMethod() {
-    super();
-    setFollowRedirects(true);
-  }
-
-  public PurgeMethod(String url) {
-    super(url);
-    setFollowRedirects(true);
-  }
+public class PurgeMethod extends HttpRequestBase {
 
   @Override
-  public String getName() {
+  public String getMethod() {
     return "PURGE";
   }
+
+  public PurgeMethod(final String uri) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)